Wordle.java

```diff
@@ -140,43 +146,18 @@ public static WordleSolver getSolver(Scanner in) throws Exception
 		}
 
 		// initialize the chosen solver
-		if (choice == 0)
-		{
-			NoDupesSolver solver = (NoDupesSolver)solvers[choice];
-			solver.initialize(loadWords(new File("words5.txt")), true);
-		}
-
-		return solvers[choice];
-	}
-
-	// =================================================================================
-
-	public static ArrayList<String> loadWords(File file) throws Exception
-	{
-		Scanner in = new Scanner(file);
-		ArrayList<String> words = new ArrayList<String>(20000);
+		WordleSolver solverChoice = solvers[choice];
+		solverChoice.initialize(loadWords("wordle.txt"));
 		
-		while (in.hasNextLine())
-		{
-			words.add(in.nextLine().strip().toLowerCase());
-		}
-
-		return words;
+		return solverChoice;
 	}
 
 	// =================================================================================
-	// use for debugging with custom lists
-	public static ArrayList<String> loadWords2(File unused)
```

Comment:
can you put loadWords2() back in? I'll never push code with it used, but it's nice for debugging